portfolio/src/main/webapp/index.html

```diff
@@ -74,6 +74,30 @@
                 <div class="page-content">
                     <p class="heading-third">Map</p>
                     <div id="map"></div>
+                    <div class="markerBox">
+                        <p id="createMarker">Create a marker!</p>
+                        <button onclick="toggleTab(createMarkerForm)">+</button>
+                        <div id="createMarkerForm">
+                        <form action="" method="POST" class="marker-form">
```

Comment:
Looks like submitting the form does not do anything. Are you planning to add code for that later? If so ideally it would be best to do that in a separate PR. But ok for now.